mod/consensus/pkg/types/slot_data.go

```diff
@@ -45,15 +45,15 @@ func (b *SlotData[AttestationDataT, SlashingInfoT]) New(
 	attestationData []AttestationDataT,
 	slashingInfo []SlashingInfoT,
 	proposerAddress []byte,
-	nextPayloadTimestamp time.Time,
+	consensusTime time.Time,
 ) *SlotData[AttestationDataT, SlashingInfoT] {
 	b = &SlotData[AttestationDataT, SlashingInfoT]{
 		slot:            slot,
 		attestationData: attestationData,
 		slashingInfo:    slashingInfo,
 		commonConsensusData: &commonConsensusData{
-			proposerAddress:      proposerAddress,
-			nextPayloadTimestamp: math.U64(nextPayloadTimestamp.Unix()),
+			proposerAddress: proposerAddress,
+			consensusTime:   math.U64(consensusTime.Unix()),
```

Comment:
just clarifying, do we need to round this before calling .Unix() ?

Reply:
or its already handled by comet

Reply:
It's handled by Unix, which picks only the second part.
See the docs
```
// Unix returns t as a Unix time, the number of seconds elapsed
// since January 1, 1970 UTC. The result does not depend on the
// location associated with t.
// Unix-like operating systems often record time as a 32-bit
// count of seconds, but since the method here returns a 64-bit
// value it is valid for billions of years into the past or future.
```